src/client/components/lazy-chat/lazy-chat.vue

```diff
@@ -12,6 +12,7 @@
       class="button button--primary lazy-chat__open-button"
       :class="{ 'lazy-chat__open-button--pending': (isAccepted && !isLoaded) }"
       @click="handleClick"
+      v-test:lazyChatButtonIdentity
```

Comment:
The reason we had "Identity" in the selector name is that the element around the logo was simply called "brand identity". You don't have to use it in every test directive. So simply `lazyChatButton` etc.